packages/front-end/pages/features/index.tsx

```diff
@@ -343,15 +345,18 @@ export default function FeaturesPage() {
                     </td>
                     <td>
                       <MoreMenu>
-                        <button
-                          className="dropdown-item"
-                          onClick={() => {
-                            setFeatureToDuplicate(feature);
-                            setModalOpen(true);
-                          }}
-                        >
-                          Duplicate
-                        </button>
+                        {permissions.check("manageFeatures", project) &&
+                        canManageFeatureDrafts ? (
```

Comment:
We should be using the feature's project, not the current project (I think `projectId` is the feature's project).  When duplicating, it copies over the project as well.

Reply:
Ah - yea, good catch.